Softeq.XToolkit.WhiteLabel/Navigation/FluentNavigators/FrameFluentNavigator.cs

```diff
@@ -15,20 +17,26 @@ namespace Softeq.XToolkit.WhiteLabel.Navigation.FluentNavigators
     public class FrameFluentNavigator<TViewModel> : FluentNavigatorBase<TViewModel> where TViewModel : IViewModelBase
     {
         private readonly IFrameNavigationService _navigationService;
+        private readonly ILogger _logger;
 
         /// <summary>
         ///     Initializes a new instance of the <see cref="FrameFluentNavigator{TViewModel}"/> class.
         /// </summary>
         /// <param name="navigationService">
         ///     The <see cref="IFrameNavigationService"/> implementation that will be used for navigation.
         /// </param>
+        /// <param name="logManager">
+        ///     The <see cref="ILogManager"/> implementation that will be used for Logger.
+        /// </param>
         /// <exception cref="T:System.ArgumentNullException">
         ///     <paramref name="navigationService"/> cannot be <see langword="null"/>.
         /// </exception>
         public FrameFluentNavigator(IFrameNavigationService navigationService)
         {
             _navigationService = navigationService
-                ?? throw new ArgumentNullException(nameof(navigationService));
+                ?? throw new ArgumentNullException(nameof(navigationService));
+
+            _logger = Dependencies.Container.Resolve<ILogManager>().GetLogger<FrameFluentNavigator<TViewModel>>();

```

Comment:
Shouldn't we inject ILogManager through constructor, especially if it is defined as one of the parameters?